src/firefly/js/visualize/ui/CatalogTableListField.jsx

```diff
@@ -202,7 +220,7 @@ class CatalogTableItem extends Component {
     render() {
         const {itemData, onClick, isClicked, cols} = this.props;
 
-        const html = '<b>' + itemData.cat[2] + '</b></br>' + htmlElementMarkup(itemData, cols);//<li key="' + `${itemData.value}` + '"> + '</li>';
+        const html = '<b>&nbsp;&nbsp;&nbsp;&nbsp;' + itemData.cat[2] + '</b></br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;' + itemMarkupTransform(itemData, cols);//<li key="' + `${itemData.value}` + '"> + '</li>';
```

Comment:
should use margin/padding instead of spaces for positioning, especially when there are so many of them.